### DISCUSSION TITLE
Historical current/tide tables

### DISCUSSION THREAD

**Discussion by TheKeeks** [jaybo/deepzoom-blog#13] [General]
Hello! Love DeepZoom. I'm trying to do some analysis on surf/fishing conditions in the Northeast. Is it possible to access historical tide (or even better, current) tables for sites viewable on DeepZoom? Thanks!
Q

**Comment by jaybo** [comment]
Here's a description of the last tide data update for the dates 2014.01.01 through 2029.12.31 inclusive.
https://blog.deepzoom.com/trip/release/tide-refresh/

**Comment by TheKeeks** [comment]
thank you!

I've never downloaded datasets like these. I am on a Mac. Ideally, I would
like a table with current strength and whether it is incoming or outgoing
at Mt. Prospect [0.6 mi SSE of (depth 15 ft), Block Island Sound, New York
Current (2022)].

Please let me know if you are able to help!

Thanks

**Reply by jaybo** [reply]
Sorry for the late reply.  Hopefully by now you've figured out that if you click the time readout at the bottom of the display you'll see the timebase.  Here you can select the starting date for the animation as well as the duration (up to one year).

![image](https://user-images.githubusercontent.com/1607323/217654228-ead31de3-517f-4f16-aee8-0b21c1d7ae90.png)
